src/utils/toLowercase.js

```diff
@@ -0,0 +1,8 @@
+// Used to lowercase string value e.g authorizerOptions.type
+export default function toLowerCase(value) {
```

Comment:
I think it would be better to avoid adding such utils and perform an in-place check that value is a string and use `value.toLowerCase()` method on it - currently it's a bit confusing as if I would see `toLowerCase` method I would expect it to accept only strings